apps/web/src/views/Swap/hooks/useFeeSaved.ts

```diff
@@ -1,11 +1,13 @@
-import { ChainId, Currency, CurrencyAmount } from '@pancakeswap/sdk'
+import { ChainId, Currency, CurrencyAmount, ERC20Token } from '@pancakeswap/sdk'
 import { DAI, USDC, USDT } from '@pancakeswap/tokens'
 import { parseNumberToFraction } from '@pancakeswap/utils/formatFractions'
 import { BigNumber } from 'bignumber.js'
 import { useCurrencyUsdPrice } from 'hooks/useCurrencyUsdPrice'
 import { useMemo } from 'react'
 
-const MAJOR_DEFINED_STABLE_COINS = {
+const MAJOR_DEFINED_STABLE_COINS: {
+  [chainId in ChainId]?: Array<ERC20Token | undefined>
```

Comment:
The list is defined by us right? So we can safely assert it's not undefined here. If any of them is undefined, the error throws on build time

```suggestion
  [chainId in ChainId]?: Array<ERC20Token>
```

Reply:
can check this [commit](https://github.com/pancakeswap/pancake-frontend/commit/06276308934e120329aa98e03500d7ad42b50a1d)

build passed with no error